packages/server/src/fhir/patient.ts

```diff
@@ -72,9 +72,7 @@ export function getPatients(resource: Resource): Reference<Patient>[] {
       }
     }
   }
-  return Array.from(result)
-    .sort((a, b) => a.localeCompare(b))
-    .map((reference) => ({ reference }));
+  return Array.from(result).map((reference) => ({ reference }));
```

Comment:
Is it intentional that the sort was removed? 👀

Reply:
There's no logical reason for this method to return sorted results.  It's sorting on `reference`, which is effectively sorting on v4 UUIDs, which is kinda arbitrary.

It does look like the tests assumed sorted results.... I will look into whether we need to update those.

Reply:
✅  tests updated